5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
```

Comment:
Btw. I'm not seeing the reason to have a separate file and not define the function directly in `common.sh`, but you might have a reason..